### PR TITLE
HTS-580: Removed get help link from govuk_wrapper

### PR DESCRIPTION
**Pull request by Jackie1988**

### CODE REVIEW

File: app/uk/gov/hmrc/helptosavefrontend/views/govuk_wrapper.scala.html [hunk 1]
```diff
@@ -49,13 +49,6 @@
 
 @headerNavLinks = {
         <li>
-            <a href="/get-help" id="get-help"
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/helptosavefrontend/views/govuk_wrapper.scala.html]: will we ever need it in the future? Is it safer to comment it out for now?

Comment (AndyHWChung) [reply app/uk/gov/hmrc/helptosavefrontend/views/govuk_wrapper.scala.html]: apparently we won't need this in the future so deleting is ok.

**Comment by AndyHWChung** [comment]
LGTM

File: app/uk/gov/hmrc/helptosavefrontend/views/govuk_wrapper.scala.html (hunk 1 above)
Comment (Jackie1988) [reply app/uk/gov/hmrc/helptosavefrontend/views/govuk_wrapper.scala.html]: Done

Comment (Jackie1988) [reply app/uk/gov/hmrc/helptosavefrontend/views/govuk_wrapper.scala.html]: oh ok

**Pull request merged**